### PR TITLE
EV Chargers

### LINKED ISSUE #183: EV Chargers
Add Electric Vehicle Chargers (ENERGY STAR Certified)

Would this go under Building/Systems or under MiscLoads?

### PR DESCRIPTION
**Pull request by nmerket**
Fixes #183 by adding the following at `HPXML/Building/BuildingDetails/Systems/ElectricVehicleChargers`

BaseElements_ElectricVehicleChargers

I'm not an expert on EV charging. What am I missing?

### CODE REVIEW

**Comment by shorowit** [comment]
How about a `NumberofUnits` field for a multifamily building that has many identical chargers?

**Comment by nmerket** [comment]
Also add charger connector

- Tesla
- SAE-J1772

File: schemas/HPXMLDataTypes.xsd [hunk 1]
```diff
@@ -1830,7 +1830,15 @@
 	<xs:simpleType name="EVChargingLevel">
 		<xs:restriction base="xs:integer">
 			<xs:minInclusive value="1"/>
-			<xs:maxInclusive value="2"/>
```
Comment (lirainer) [inline schemas/HPXMLDataTypes.xsd]: @nmerket Did you mean to drop level 2?

Comment (lirainer) [reply schemas/HPXMLDataTypes.xsd]: Sorry, read right through that 🙄 . Was thinking it was an enumeration.

**Comment by nmerket** [comment]
Add `ConnectedDevice` from #198.

**Comment by nmerket** [comment]
This should be merged _after_ #198.

**Comment by shorowit** [comment]
@nmerket Are you purposefully not adding a `NumberofUnits` element? I can't remember if this was discussed, but I had mentioned it above.

**Comment by nmerket** [comment]
@shorowit No, I just forgot.

**Pull request merged**